rest-api/tools/csv_importer.py

```diff
@@ -21,6 +24,7 @@ def __init__(self, entity_name, dao, id_field, external_id_field, required_colum
     self.external_id_field = external_id_field
     self.required_columns = required_columns
     self.errors = list()
+    self.new_sites_list = []
```

Comment:
This (and all other changes to this file) belong in SiteImporter, not here.

You also aren't adding anything to new_sites_list AFAICT.